tools/api-report/README.md

```
@@ -0,0 +1,136 @@
+
+```bash
+pnpm api-report:fix
+pnpm nx run-many -t build -p journey-client oidc-client device-client davinci-client
+pnpm api-report
+```
+
+### Pre-commit Failure Output
+
+When the tool fails in the lefthook pre-commit hook, it shows the specific missing symbols and the exact commands to fix them:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)